src/WebExternalSearchProvider.cs

```diff
@@ -240,6 +239,39 @@ public IPreviewImage GetPrimaryEntityPreviewImage(ExecutionContext context, IExt
             return null;
         }
 
+        public ConnectionVerificationResult VerifyConnection(ExecutionContext context, IReadOnlyDictionary<string, object> config)
+        {
+            var client = new RestClient("https://google.com");
```

Comment:
This should not be hardcoded to google.com . It shouldn't this use the url the user provided